### PR TITLE
a-tiny-JS-world

### PR DESCRIPTION
**Pull request by mazvira**
### a-tiny-JS-world

[Demo](https://mazvira.github.io/a-tiny-JS-world) |
[Code base](https://github.com/mazvira/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/mazvira/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+print("I am:" + man.species + ';' + "My name is:" + man.name + ';' + "My gender is:" + man.gender + ';' + "I have " + man.legs + " legs" + ';'
++ "I have " + man.hands + " hands" + ';' + "My favorite saying is:" + man.saying + ';'  + "My friends are:" + man.friends());
+
+print("I am:" + catWoman.species + ';' + "My name is:" + catWoman.name + ';' + "My gender is:" + catWoman.gender + ';' + "I have " + catWoman.legs + " legs" + ';'
++ "I have " + catWoman.hands + " hands" + ';' + "My favorite saying is:" + catWoman.saying + ';'  + "My friends are:" + catWoman.friends());
+
+cat.setSaying("meooow!");
+print("Cat`s new saying is: " + cat.saying + " and " + "CatWoman`s new saying is: " + catWoman.saying);
+
+
```
Comment (OleksiyRudenko) [inline submissions/mazvira/a-tiny-JS-world/index.js]: There is also a series of `print` calls.
Let's improve employing `Array.forEach`
A hint: `[varA, varB].forEach(item => whateverFunction(whateverAnotherFunc(item)));`

**Comment by OleksiyRudenko** [comment]
@mazvira Please, change the PR title so it gave better idea of what's inside right from the PRs lisiting.
Task name would be a perfect title.
image

**Comment by mazvira** [comment]
I have changed, thank you!

File: submissions/mazvira/a-tiny-JS-world/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    name: 'Sergejs',
+    gender: 'male',
+    legs: 2,
+    hands: 2,
+    saying: 'Hello world!',
+    friends : ["Toby", "Tom", "Olena"]
+};
+
+function printInhabitants(obj){
+    return Object.values(obj).join(" - ");
```
Comment (OleksiyRudenko) [inline submissions/mazvira/a-tiny-JS-world/index.js]: The order of properties in object is not guaranteed.
Also a different contributor may add yet another inhabitant and specify props in a different order.
Please, list properties explicitly.

A quote form one of past comments:

> A hint:
```
[
 variableA,
 "text:,
 `some text: ${variableB}`,
 someFunctionCall(),
].join(delimiter);
```

Comment (mazvira) [reply submissions/mazvira/a-tiny-JS-world/index.js]: changed, thanks!

**Pull request merged**